pkg/dao/repository.go

```diff
@@ -100,10 +100,11 @@ func (r repositoryDaoImpl) List(
 
 	if filterData.Version != "" {
 		versions := strings.Split(filterData.Version, ",")
-		filteredDB = filteredDB.Where("? = any (versions)", versions[0])
+		orGroup := r.db.Where("? = any (versions)", versions[0])
 		for i := 1; i < len(versions); i++ {
-			filteredDB.Or("? = any (versions)", versions[i])
```

Comment:
would: 
filteredDB = filteredDB.Or("? = any (versions)", versions[i]) 
not work here?

Reply:
I'm happy to keep it with your method, just curious.